lkmpg.tex

```diff
@@ -174,6 +174,20 @@ \subsection{Before We Begin}
         You will not see it unless you look through your \sh|journalctl| .
         See \ref{sec:helloworld} for details.
         To have immediate access to this information, do all your work from the console.
+
+  \item SecureBoot.
+        The modern laptop or PC are shipped with UEFI that has enabled the SecureBoot by default. And some Linux distributions are shipped with kernels that require the kernel modules to be signed with some security keys. If you cannot inssert the kernel module with inmod:
+
+\begin{codebash}
+insmod ./hello-1.ko
+insmod: ERROR: could not insert module ./hello-1.ko: Operation not permitted
```

Comment:
Move the message "ERROR: could not insert module ./hello-1.ko: Operation not permitted" out of `codebash` block.